lib/manageiq/providers/workflows/builtin_runner.rb

```diff
@@ -0,0 +1,62 @@
+module ManageIQ
+  module Providers
+    module Workflows
+      require "floe"
+
+      class BuiltinRunnner < Floe::Runner
+        SCHEME = "builtin"
+
+        def run_async!(resource, params = {}, secrets = {}, context = {})
+          scheme_prefix = "#{SCHEME}://"
+          raise ArgumentError, "Invalid resource" unless resource&.start_with?(scheme_prefix)
+
+          method = resource.sub(scheme_prefix, "")
+
+          # TODO: prevent calling anything except the specifics methods, e.g. you shouldn't be able to call .to_s.
+          #       Maybe make BuiltinMethods a BasicObject?
+
+          builtin_method_kwargs = params.transform_keys { |k| k.downcase.to_sym }
+
+          BuiltinMethods.public_send(method, **builtin_method_kwargs)
```

Comment:
🤔 I wonder if this is doing too much work in the base.  Thinking ahead, when we get to the provision task thing, we're going to want information from the execution context.  I wonder if it's better to just give the "raw" params and context to the method and let it deal with the details